crates/proto/src/version/v671/helper.rs

```diff
@@ -0,0 +1,8 @@
+use crate::helper::ProtoHelper;
+use crate::version::v729::gamepackets::GamePackets;
+
+pub struct ProtoHelperV671;
```

Comment:
isnt this PR for `766` and not `671`?

Reply:
Started doing v671 before just doing v766